### ISSUE TITLE
Mismatch between logging configuration in appsettings.json and appsettings.development.json

### ISSUE THREAD

**Issue by mjrousos** [aspnet/templating#316]
In a new ASP.NET Core web api application (created with `dotnet new webapi` with .NET CLI v. 2.1.4), running in development mode does not show informational/debug logs as expected.

The problem is that the logging configuration in appsettings.json is in a different format (Logging.[Debug|Console].LogLevel) than appsettings.Development.json (Logging.LogLevel.Default), so the settings in appsettings.Development.json don't effectively override the settings in appsettings.json as they're supposed to.

**Expected Behavior**: When running an ASP.NET Core web API from the new project template, more verbose logging should happen when in Development mode.

**Actual Behavior**: Enabling development mode for the ASP.NET Core web API does not increase logging verbosity.

**Comment by Eilon** [comment]
@mjrousos thanks for the report, we will take a look.

**Comment by Eilon** [comment]
@mjrousos I took a closer look at this and I'm not sure I see the behavior you're describing.

Here are the default files in the project template:

### `appsettings.json`

```json
{
  "Logging": {
    "IncludeScopes": false,
    "LogLevel": {
      "Default": "Warning"
    }
  }
}
```

### `appsettings.Development.json`

```json
{
  "Logging": {
    "IncludeScopes": false,
    "LogLevel": {
      "Default": "Debug",
      "System": "Information",
      "Microsoft": "Information"
    }
  }
}
```

The two files have the same format as best I can tell, but the Development one overrides the "Default" log level, and then sets a couple of other explicit log levels.

Can you share more info on what you're seeing?

**Comment by mjrousos** [comment]
Thanks, @Eilon, for taking a look. I agree those files look correct. Here's what I'm seeing after running `dotnet new webapi`:

**appsettings.json**
```json
{
  "Logging": {
    "IncludeScopes": false,
    "Debug": {
      "LogLevel": {
        "Default": "Warning"
      }
    },
    "Console": {
      "LogLevel": {
        "Default": "Warning"
      }
    }
  }
}
```

**appsettings.Development.json**
```json
{
  "Logging": {
    "IncludeScopes": false,
    "LogLevel": {
      "Default": "Debug",
      "System": "Information",
      "Microsoft": "Information"
    }
  }
}
```

Experimenting some more, though, I get the same files as you if I run `dotnet net mvc`. I think there must be a difference between the *webapi* and *mvc* templates.

**Comment by mjrousos** [comment]
I dug around and it looks like [this](https://github.com/aspnet/templating/blob/release/2.0/src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-CSharp/appsettings.json) may be the offending file. It appears to be [fixed](https://github.com/aspnet/templating/commit/e5a9bd3dd7407b28eb568906cd505f8740a5e132) in the dev branch, though, so perhaps that means the issue is already addressed going forward.

**Comment by Eilon** [comment]
Ah, yes, indeed it appears to be fixed in 2.1, so closing this issue.
